packages/hub-nodejs/src/generated/message.ts

```diff
@@ -1526,6 +1569,93 @@ export const ReactionBody = {
   },
 };
 
+function createBaseTagBody(): TagBody {
+  return { value: "", targetCastId: undefined, targetUrl: undefined };
+}
+
+export const TagBody = {
+  encode(message: TagBody, writer: _m0.Writer = _m0.Writer.create()): _m0.Writer {
+    if (message.value !== "") {
+      writer.uint32(10).string(message.value);
+    }
+    if (message.targetCastId !== undefined) {
+      CastId.encode(message.targetCastId, writer.uint32(18).fork()).ldelim();
+    }
+    if (message.targetUrl !== undefined) {
+      writer.uint32(26).string(message.targetUrl);
+    }
+    return writer;
+  },
+
+  decode(input: _m0.Reader | Uint8Array, length?: number): TagBody {
+    const reader = input instanceof _m0.Reader ? input : _m0.Reader.create(input);
+    let end = length === undefined ? reader.len : reader.pos + length;
+    const message = createBaseTagBody();
+    while (reader.pos < end) {
+      const tag = reader.uint32();
+      switch (tag >>> 3) {
+        case 1:
+          if (tag != 10) {
+            break;
+          }
+
+          message.value = reader.string();
+          continue;
+        case 2:
+          if (tag != 18) {
+            break;
+          }
+
+          message.targetCastId = CastId.decode(reader, reader.uint32());
+          continue;
+        case 3:
+          if (tag != 26) {
+            break;
+          }
+
+          message.targetUrl = reader.string();
+          continue;
+      }
+      if ((tag & 7) == 4 || tag == 0) {
+        break;
+      }
+      reader.skipType(tag & 7);
+    }
+    return message;
+  },
+
+  fromJSON(object: any): TagBody {
+    return {
+      value: isSet(object.value) ? String(object.value) : "",
+      targetCastId: isSet(object.targetCastId) ? CastId.fromJSON(object.targetCastId) : undefined,
+      targetUrl: isSet(object.targetUrl) ? String(object.targetUrl) : undefined,
+    };
+  },
+
+  toJSON(message: TagBody): unknown {
+    const obj: any = {};
+    message.value !== undefined && (obj.value = message.value);
+    message.targetCastId !== undefined &&
+      (obj.targetCastId = message.targetCastId ? CastId.toJSON(message.targetCastId) : undefined);
+    message.targetUrl !== undefined && (obj.targetUrl = message.targetUrl);
+    return obj;
+  },
+
+  create<I extends Exact<DeepPartial<TagBody>, I>>(base?: I): TagBody {
+    return TagBody.fromPartial(base ?? {});
+  },
+
+  fromPartial<I extends Exact<DeepPartial<TagBody>, I>>(object: I): TagBody {
+    const message = createBaseTagBody();
+    message.value = object.value ?? "";
+    message.targetCastId = (object.targetCastId !== undefined && object.targetCastId !== null)
+      ? CastId.fromPartial(object.targetCastId)
+      : undefined;
+    message.targetUrl = object.targetUrl ?? undefined;
+    return message;
+  },
+};
+
```

Comment:
cool, so this is all auto-genned changes for the new msg type? Nothing added by hand here?

Reply:
yes, that is the case for everything under `/generated`